crates/swc_ecma_parser/src/parser/stmt.rs

```diff
@@ -1173,6 +1173,20 @@ impl<'a, I: Tokens> Parser<I> {
 
         let init = if eat_exact!(self, ';') {
             return self.parse_normal_for_head(None);
+        } else if is!(self, "async") && peeked_is!(self, "of") {
```

Comment:
I am trying to look forward two words. Is there any way to do this? @kdy1

Reply:
No, it's not possible at now